src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class Fruit {
+    private String name;
+    private int quantity;
+
+    public Fruit(String name, int quantity) {
+        Objects.requireNonNull(name);
+        checkForNegativeQuantity(quantity);
+        this.name = name;
+        this.quantity = quantity;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        Objects.requireNonNull(name);
+        this.name = name;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        checkForNegativeQuantity(quantity);
+        this.quantity = quantity;
+    }
+
+    private void checkForNegativeQuantity(int quantity) {
+        if (quantity < 0) {
+            throw new RuntimeException("Value is negative");
+        }
+    }
```

Comment:
same, as well as null check

Reply:
I removed FruitHandler => i removed these methods into the class Fruit